AnnoyancesFilter/sections/popups.txt

```diff
@@ -372,6 +372,8 @@ mojanorwegia.pl#$#.modMnNewLayerBack { display: none !important; }
 mojanorwegia.pl#$#.filter-blur { filter: none !important; }
 mojanorwegia.pl#$#body { overflow: visible !important; }
 vanityfair.com##.persistent-bottom
+fastbikesmag.com#$#.hustle-popup { display: none !important; }
+fastbikesmag.com#%#//scriptlet('remove-class', 'hustle-no-scroll', 'html')
```

Comment:
Ad-leftover at the right, newsletter and social widget at the left - please block them